openvidu-server/src/main/java/io/openvidu/server/core/SessionEventsHandler.java

```diff
@@ -164,6 +167,19 @@ public void onParticipantJoined(Participant participant, String sessionId, Set<P
 				openviduBuildConfig.getOpenViduServerVersion());
 		result.addProperty(ProtocolElements.PARTICIPANTJOINED_MEDIASERVER_PARAM,
 				this.openviduConfig.getMediaServer().name());
+		switch (this.openviduConfig.getMediaServer()) {
+			case mediasoup:
+				// mediasoup supports simulcast. Enable it if allowed by session configuration.
+				result.addProperty(ProtocolElements.PARTICIPANTJOINED_SIMULCAST_PARAM,
+						this.sessionManager.getSessionWithNotActive(sessionId).getSessionProperties()
```

Comment:
Access here the current Session's *allow* setting, to tell the client whether it is allowed or not to enable Simulcast.

I'm not sure if `getSessionWithNotActive` is the correct method to call here.